privileges/privileges_linux.go

```diff
@@ -0,0 +1,34 @@
+//go:build linux || unix
+
+package privileges
+
+import (
+	"os"
+	"runtime"
+
+	"github.com/projectdiscovery/naabu/v2/pkg/israce"
+	"golang.org/x/sys/unix"
+)
+
+// isPrivileged checks if the current process has the CAP_NET_RAW capability or is root
+func isPrivileged() bool {
```

Comment:
Since this is now an import library, I think this method should only check for `CAP_NET_RAW` capability and should be renamed accordingly